Blankmon/src/application/AnatureBuilder.java

```diff
@@ -166,7 +168,7 @@ private MoveSet generateMoveSet(Species species, int level)
 		return new MoveSet(move1, move2, move3, move4);
 	}
 
-	private int generateRandomUniqueIndex(int max, int toGenerate, int otherIndex1, int otherIndex2, int otherIndex3)
+	private static int generateRandomUniqueIndex(int max, int toGenerate, int otherIndex1, int otherIndex2, int otherIndex3)
 	{
 		Random r = new Random();
 		toGenerate = r.nextInt(max);
```

Comment:
Ah, I see we changed these to static so we don't have to create an instance. Clever!